### PR TITLE
fix: changes trivia to interview question in all learn folders

### PR DESCRIPTION
**Pull request by Hrithik0112**
## Fixes Issue
- Fixes #59 #49 

## Changes

changes "trivia" name to "Interview Question" in All "Learn" folder i.e. Html basic ,CSS basic , CSS flex , CSS gird.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Screenshot 2023-10-02 112205

Screenshot 2023-10-02 112137
Screenshot 2023-10-02 112217
endFreaks/Official-Website/assets/122002784/e4621dc9-9c3e-4da8-9d49-687a8c840c48)

### CODE REVIEW

File: content/batch/learn/css/flex.mdx [hunk 1]
```diff
@@ -33,7 +33,9 @@ Want to improve this page?. Raise a issue on [@github](https://github.com/Manish
 <TabsList>
   <TabsTrigger value="learn">Learn</TabsTrigger>
   <TabsTrigger value="assignment">Assignment</TabsTrigger>
-  <TabsTrigger value="InterviewQuestion">Interview Question</TabsTrigger>
+
```
Comment (Vishal-raj-1) [inline content/batch/learn/css/flex.mdx:36]: remove extra line

File: content/batch/learn/css/flex.mdx [hunk 2]
```diff
@@ -200,10 +202,12 @@ Want to improve this page?. Raise a issue on [@github](https://github.com/Manish
 
 </TabsContent>
 
-<TabsContent value="InterviewQuestion">
+<TabsContent value="interview">
+
```
Comment (Vishal-raj-1) [inline content/batch/learn/css/flex.mdx:206]: remove extra line

File: content/batch/learn/css/grid.mdx [hunk 3]
```diff
@@ -187,6 +188,11 @@ Want to improve this page?. Raise a issue on [@github](https://github.com/Manish
    <VideoPlayer src="https://youtu.be/ULp7wPJ-rzQ"/>
 </TabsContent>
 
+<TabsContent value="interview">
+    Interview Questions on css grid
```
Comment (Vishal-raj-1) [inline content/batch/learn/css/grid.mdx:192]: it should be "CSS Grid"

File: content/batch/learn/html/basic.mdx [hunk 4]
```diff
@@ -299,6 +301,11 @@ Want to improve this page?. Raise a issue on [@github](https://github.com/Manish
     Interview Questions on HTML
 </TabsContent>
 
+<TabsContent value="interview">
+    Interview Questions on html
```
Comment (Vishal-raj-1) [inline content/batch/learn/html/basic.mdx:305]: it should be "HTML"

File: content/batch/learn/html/basic.mdx [hunk 5]
```diff
@@ -20,7 +20,9 @@ Want to improve this page?. Raise a issue on [@github](https://github.com/Manish
 <TabsList>
   <TabsTrigger value="learn">Learn</TabsTrigger>
   <TabsTrigger value="assignment">Assignment</TabsTrigger>
-  <TabsTrigger value="InterviewQuestion">Interview Question</TabsTrigger>
+
```
Comment (Vishal-raj-1) [inline content/batch/learn/html/basic.mdx:23]: remove extra line

File: content/batch/learn/css/grid.mdx [hunk 6]
```diff
@@ -187,6 +188,11 @@ Want to improve this page?. Raise a issue on [@github](https://github.com/Manish
...
+</TabsContent>
+
```
Comment (Vishal-raj-1) [inline content/batch/learn/css/grid.mdx:194]: remove extra line

File: content/batch/learn/css/basic.mdx [hunk 7]
```diff
@@ -59,6 +60,10 @@ Want to improve this page?. Raise a issue on [@github](https://github.com/Manish
 
 </TabsContent>
 
+<TabsContent value="interview">
+    Interview Questions on css
```
Comment (Vishal-raj-1) [inline content/batch/learn/css/basic.mdx:64]: it should be "CSS"

**Review by Vishal-raj-1 — CHANGES_REQUESTED** [review]
fix all changes

**Comment by Hrithik0112** [comment]
Understood.

**Comment by Hrithik0112** [comment]
@Vishal-raj-1 done as you told. is there anything to improve ?

**Pull request merged**